Fandom Creator/JavaScript.md

```diff
@@ -34,8 +38,37 @@ class Toggle extends React.Component {
 }
 ```
 
+## Instance fields in a class
+
+**Requirement Level:** MUST
```

Comment:
What I meant earlier is that we should be using MUST/SHOULD/MAY within the actual paragraph text to allow for more nuance for each section. That way we can use multiple MUST/SHOULD/MAYs as needed within each paragraph. So below where you say:

'Fields must be declared as "instance fields" over setting them in the constructor (even when a constructor is required for other reasons).'

It would instead read:
'Fields MUST be declared as "instance fields" over setting them in the constructor (even when a constructor is required for other reasons).'

Then for example you could have another paragraph in this section that used MAY or SHOULD, but it wouldn't conflict with the paragraph above. What I'm really suggesting is that this document be descriptive rather than proclamatory in nature.